pkg/redis/codec/main.go

```diff
@@ -0,0 +1,78 @@
+// Copyright © 2020 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package codec provides a codec, which encodes and decodes protocol buffers
+// stored in Redis to/from JSON.
+package main
+
+import (
+	"flag"
+	"io/ioutil"
+	"log"
+	"os"
+
+	"github.com/gogo/protobuf/proto"
+	"go.thethings.network/lorawan-stack/v3/pkg/jsonpb"
+	"go.thethings.network/lorawan-stack/v3/pkg/redis"
+	"go.thethings.network/lorawan-stack/v3/pkg/ttnpb"
+)
+
+var json = jsonpb.TTN()
+
+func init() {
+	log.SetOutput(os.Stderr)
+	log.SetFlags(0)
+}
+
+func main() {
+	typ := flag.String("type", "", "Proto type to be used (supported: ttnpb.EndDevice)")
+	encode := flag.Bool("encode", false, "Whether encoding should be performed (default: false)")
```

Comment:
Afaik the default is automatically printed by `Usage()`

Reply:
I had the same impression, but apparently it's not:
```
go run ./pkg/redis/codec -h          
Usage of /tmp/go-build797321260/b001/exe/codec:
  -encode
    	Whether encoding should be performed (default: false)
  -type string
    	Proto type to be used (supported: ttnpb.EndDevice)
exit status 2
```